packages/validation/src/routes/applicationRoutes.ts

```diff
@@ -62,3 +62,11 @@ export const editApplicationRequestSchema = z.object({
 		params: { violation: 'noEmptyObject' },
 	}),
 });
+
+export const submitApplicationRequestSchema = z
+	.object({
+		applicationId: z.string().min(1),
+		role: z.enum(['APPLICANT', 'REP']),
+		signature: z.string().min(1),
```

Comment:
The signature is confirmed to always be a base64 Image so adding the same validation we have in the `signatureRoutes.ts` file in the validation package makes sense here. Also, for role (called `signee` in the sig API), lets make it consistent and use INSTITUTIONAL_REP. 

Also given this is being passed through the body, appID will be a number and can be validated (unlike in the params validation).

```suggestion
	.object({
		applicationId: z.number().nonnegative(),
	      role: z.literal('APPLICANT').or(z.literal('INSTITUTIONAL_REP')),
	      signature: z.string().regex(BASE64_IMAGE),
```

Reply:
Great suggestion! Updated the code now..